documentation/airport-guides/lkpr.md

```diff
@@ -0,0 +1,66 @@
+
+It serves the capital city and most of the area west of Prague. In the east it is helped by Brno Tuřany (LKTB), Pardubice Airport (LKPD) and Ostrava Mošnov (LKMT). It is located 7NM (13 km) to the North-West of the city centre. 
+
+The Czech civil aviation history started at Prague Kbely Airport in 1918, but it's capacity was insufficient by the mid 1930s the Czech government decided to build a new airport in Prague-Ruzyně. Throughout the years, the airport went through many developmental changes, mainly the construction of runways 04/22, 06/24 and 12/30. 
+
+### Statistics
+
+15,415,001 passengers in 2017
+148,283 aircraft movement in 2017
+
+[Source](https://en.wikipedia.org/wiki/V%C3%A1clav_Havel_Airport_Prague)
+
+### Airspace
+
+The airspace is mainly a circle with a radius of 35nm situated around the Václav Havel Airport with two extended pieces of the airspace to the South-East and South-West. It stretches from ground level to FL145, although departures are permitted to climb past that altitude. 
+
+### Standard Operations
+
+###### Runway usage
+
+The Václav Havel Airport is very specific with it's runway usage. RWY04/22 is permanently out of operation and serves as a taxiway for oversized aircraft such as the Antonov AN-225. It is also for this reason not at all included in openScope. RWY24 is the only runway approved for ILS CAT II/III operations, runways 06, 12 and 30 are ILS CAT I certified. 
+
+The monopoly in CAT III certification and winds coming mostly from the west mean RWY24 is the preffered RWY and is used for approximately 80% of all operations. RWY12 is the second most commonly used runway and RWYs 06 and 30 are used very seldom. RWYs 06/24 and 12/30 are <b>never</b> used simultaneously. 
+
+### Instrument Procedures
+
+###### Departures
+
+A maximum speed of 250kts should be maintained below FL100, however it is common practice for Praha Radar to allow aircraft to fly past those restrictions.
+
+###### Arrivals
+
+There are established STAR procedures with speed and altitude restrictions, however aircraft are often vectored for the approach to the runway in use. 
+
+### Airports of interest
+
+##### LKPR
+Mezinárodní letiště Václava Havla Praha
+148,283 aircraft movement in 2017
+
+##### LKVO
+Letiště Vodochody
+
+Vodochody Airport is a small GA airport located 9NM to the North-East of LKPR. It has one runway, 10/28, with a length of 8202ft and an emergency strip at a length of 4593ft. There are no designed SIDs for this airport, all traffic is IFR and vectored out of the airport. 
+
+##### LKKB
+Letiště Praha-Kbely
+
+Prague Kbely Airport is a military airport located 11NM to the East of LKPR. It was built as the first international airport in the Czech Republic in 1918, however it was soon understood that it's capacity is much smaller then neccessary and all commercial air traffic was moved to the Václav Havel Airport by the late 1930s. It has one runway, 06/24, with a length of 6824ft. 
```

Comment:
replace with
Prague Kbely Airport is a military airport located 11nm to the East of LKPR. It was built as the first international airport in the Czech Republic in 1918, however, it was soon understood that its capacity is much smaller then necessary and all commercial air traffic was moved to the Václav Havel Airport by the late 1930s. It has one runway, 06/24, with a length of 6824ft.